### PR TITLE
Fix ScreenshotTest

### PR DESCRIPTION
**Pull request by irfano**
Closes: woocommerce/woomobile-private#393

### Description

This fixes failing `ScreenshotTest` and related script in `Fastfile`. 

### Steps to reproduce

1. Run `ScreenshotTest`.
2. Run `bundle exec fastlane take_screenshotslocales:en-US`.

#### ScreenshotTest
The test should pass.

#### bundle exec fastlane take_screenshotslocales:en-US
This command should generate screenshots in `/fastlane/screenshots/raw/en-US/images/phoneScreenshots`

### The tests that have been performed

Generated raw screenhots.
image

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [x] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.
- [ ] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.
- [x] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

### CODE REVIEW

File: WooCommerce/src/androidTest/assets/mocks/mappings/jetpack-blogs/wc/system_status.json [hunk 1]
```diff
@@ -69,8 +69,8 @@
                 }, {
                     "plugin": "woocommerce-gateway-stripe/woocommerce-gateway-stripe.php",
                     "name": "WooCommerce Stripe Gateway",
-                    "version": "6.1.0",
-                    "version_latest": "6.1.0",
+                    "version": "9.9.9",
+                    "version_latest": "9.9.9",
```
Comment (irfano) [inline WooCommerce/src/androidTest/assets/mocks/mappings/jetpack-blogs/wc/system_status.json:73]: I updated these versions to the max to prevent tests from failing due to min version requirements.

File: WooCommerce/src/androidTest/assets/mocks/mappings/woopos/stripe_account.json [hunk 2]
```diff
@@ -19,9 +19,10 @@
     "jsonBody": {
       "data": {
         "country": "US",
-        "storeCurrencies": {
+        "store_currencies": {
```
Comment (irfano) [inline WooCommerce/src/androidTest/assets/mocks/mappings/woopos/stripe_account.json:22]: This change is unrelated to this PR and doesn’t affect anything, but I corrected it to match the actual model.

File: WooCommerce/src/androidTest/kotlin/com/woocommerce/android/di/MockCardReaderManagerModule.kt [hunk 3]
```diff
@@ -42,7 +42,7 @@ class MockCardReaderManagerModule {
             override val id: String
                 get() = "ADEE123"
             override val type: String
-                get() = "reader"
+                get() = "COTS_DEVICE"
```
Comment (irfano) [inline WooCommerce/src/androidTest/kotlin/com/woocommerce/android/di/MockCardReaderManagerModule.kt:45]: I changed "reader" since it's an undefined type and causes a crash.

File: WooCommerce/src/androidTest/kotlin/com/woocommerce/android/e2e/screens/orders/OrderListScreen.kt [hunk 4]
```diff
@@ -15,8 +15,7 @@ import org.hamcrest.Matchers
 
 class OrderListScreen : Screen(R.id.ordersList) {
     fun selectOrder(index: Int): SingleOrderScreen {
-        val correctedIndex = index + 1 // account for the header
```
Comment (irfano) [inline WooCommerce/src/androidTest/kotlin/com/woocommerce/android/e2e/screens/orders/OrderListScreen.kt:18]: There are now two headers in the mock data, so I removed header handling from this function; it will be managed in the calling location.

**Review by JorgeMucientes — APPROVED** [review]

**Comment by JorgeMucientes** [comment]
Great job fixing this @irfano! Works well. I could run the `Screenshots` class successfully. However, as discussed over DM I was not able to run `bundle exec fastlane take_screenshotslocales:en-US` due to my local setup. Since the main changes belong to the `Screenshots` class and related, not being able to run the fastlane command shouldn't be an issue. 
Let's :shipit:
